homeassistant/components/jellyfin/media_source.py

```diff
@@ -158,7 +158,10 @@ async def _build_library(
         if collection_type == COLLECTION_TYPE_TVSHOWS:
             return await self._build_tv_library(library, include_children)
 
-        raise BrowseError(f"Unsupported collection type {collection_type}")
+        # Unsupported collection types have already been filtered out in library retrieval
+        raise BrowseError(
+            f"Unsupported collection type {collection_type}"
+        )  # pragma: no cover
```

Comment:
`# pragma: no cover` is not allowed here.

Either it is something that can never happen, is dead code, and can be removed; or it is something that just doesn't have a test yet.

Reply:
It was dead code indeed. I have removed the statement.